src/views/TradingCompetition/MoboxCompetition.tsx

```diff
@@ -48,23 +45,28 @@ const MoboxCompetition = () => {
   const profileApiUrl = process.env.NEXT_PUBLIC_API_PROFILE
   const { account } = useWeb3React()
   const { t } = useTranslation()
-  const { profile, isLoading } = useProfile()
+  const { profile, isLoading: isProfileLoading } = useProfile()
   const { isDark, theme } = useTheme()
   const tradingCompetitionContract = useTradingCompetitionContractMobox(false)
-  const [currentPhase, setCurrentPhase] = useState(CompetitionPhases.REGISTRATION)
+  const [currentPhase, setCurrentPhase] = useState(() => {
```

Comment:
This will set the state statically, if users date time is wrong phase will be corrected after fetchin onchain data